docs/reference/logging/fluentd.md

```diff
@@ -34,7 +34,8 @@ Some options are supported by specifying `--log-opt` as many times as needed:
 
  - `fluentd-address`: specify `host:port` to connect `localhost:24224`
  - `tag`: specify tag for fluentd message, which interpret some markup, ex `{{.ID}}`, `{{.FullID}}` or `{{.Name}}` `docker.{{.ID}}`
-
+ - `ignore-startup-connect-error`: true/false; Should the logging driver ignore the error on container startup and reconnect in the background. Default: false
```

Comment:
@tiborvass `reconnect-on-startup-error` sounds good to me. Shall we lock that down as the final answer? :)